docker-integration-tests/pom.xml

```diff
@@ -34,6 +34,13 @@
     <sbt.project.name>docker-integration-tests</sbt.project.name>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>db2</id>
+      <url>https://app.camunda.com/nexus/content/repositories/public/</url>
```

Comment:
This this repository going to stay around indefinitely? I don't like depending on things which aren't in Maven Central since we've been bitten in the past by repositories changing artifacts, disappearing, or having uptime problems.